UnitySDK/Assets/ML-Agents/Editor/Tests/RayPerceptionTests.cs

```diff
@@ -10,7 +10,7 @@ public class RayPerceptionTests : MonoBehaviour
         public void TestPerception3D()
         {
             var angles = new[] {0f, 90f, 180f};
-            var tags = new[] {"test", "test_1"};
+            var tags = new[] {"wall", "pit"};
```

Comment:
I do not like the idea of introducing more example environment specific code into the Core code of ML-Agents.
I am not up to speed on this. What is this solving?

Reply:
If an example scene is open, the test requires tags which are actually part of the project, or it will fail. As such we are not relying on the example environment code, but we are relying on the project's defined tags. It is unclear to me how to check for tags without having them be actually defined. It is possible to edit the tag list using a script, but this seems to make other undesirable assumptions about the structure of the project.

Reply:
Why are we running the unit-tests with the example scenes? Can we just open a new scene when doing testing. I really don't think we should have tests that rely on project settings.

Reply:
I ran into an issue where I had a scene open and ran the unit tests.  The ray cast perception tests ended up hitting something and went down an untested code path which uses `CompareTag`.  This caused the test to fail.  If anything, I think we should have playmode tests for these kinds of tests where we have a test scene and do a full test of things like ray cast perception.

Reply:
Happy to close this PR if we want to go with a more thorough solution instead. I figured at the very least this change would cause slightly less pain for people trying to run the Unity tests as they currently are set up.